narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -365,13 +397,13 @@ function process_zip_list_file {
 
 		check_if_tag_exists liferay-dxp "${tag_name_new}" && continue
 
-		check_ignore_zip_file "${hotfix_zip_file}" "${release_version}" && continue
+		check_ignore_zip_file "${hotfix_zip_file}" && continue
 
-		file_url="${zip_directory_url}/${hotfix_zip_file}"
+		zip_file_path="${zip_directory_url}/${hotfix_zip_file}"
```

Comment:
Why did you change the variable name?

Reply:
You said that file_url is not the best for it and it should be something with "path"